openedx/adg/lms/course_meta/admin.py

```diff
@@ -0,0 +1,18 @@
+"""
+Admin configurations for course_meta app
+"""
+from django.contrib import admin
+
+from .models import CourseMeta
+
+
+class CourseMetaAdmin(admin.ModelAdmin):
+    """
+    Django admin customizations for CourseMeta model
+    """
+
+    list_display = ('course_id', 'is_prereq')
+    search_fields = ('course_id',)
+
+
+admin.site.register(CourseMeta, CourseMetaAdmin)
```

Comment:
How about using the [the-register-decorator](https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#the-register-decorator) to bind model with ModelAdmin class. 
It will give 2 benefits, 

1. If we bind multiple models with multiple admins, it will not separte register line away from ModelAdmin class. 
2. We don't need to repeat class name to register model with ModelAdmin

Reply:
Good suggestion!